_includes/career-profile.html

```diff
@@ -5,6 +5,7 @@ <h2 class="section-title"><i class="fa fa-user"></i>{{ site.data.data.career-pro
 
     <div class="summary">
       <p>{{ site.data.data.career-profile.text }}</p>
+      <p>{{ site.data.data.career-profile.additional }}</p>
```

Comment:
Not against but just `<br><br>` would give the same effect. And based on that it is probably unnecessary change.

Reply:
Gotcha, I just wasn't sure how to do a newline via the templating engine (e.g. tried '\n') -- might be worth adding a note in the docs or comments?

Reply:
Well, I think probably the best way is, probably, to do it as the default example.

Reply:
Removed the additional property and put a newline using <br> tags in the data sample (as a demonstration).